<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaTeX Office Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="DocumentViewer" name="documentViewer" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionRun_Slide_Show"/>
    <addaction name="actionUnpackage_Document"/>
    <addaction name="separator"/>
    <addaction name="actionCompiler_Settings"/>
    <addaction name="actionRemote_Control_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionLaTeX_Office_Viewer_Help"/>
    <addaction name="actionLaTeX_Office_on_the_Web"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMinimize_window"/>
    <addaction name="actionMaximize"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionMinimize_Screen"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/open.png</normaloff>:/images/menu/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/preview.png</normaloff>:/images/menu/preview.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/print.png</normaloff>:/images/menu/print.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;rint...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/search.png</normaloff>:/images/menu/search.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLaTeX_Office_Viewer_Help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/help.png</normaloff>:/images/menu/help.png</iconset>
   </property>
   <property name="text">
    <string>LaTeX Office Viewer &amp;Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+'</string>
   </property>
  </action>
  <action name="actionAbout_LaTeX_Office_Viewer">
   <property name="text">
    <string>&amp;About LaTeX Office Viewer</string>
   </property>
  </action>
  <action name="actionCompiler_Settings">
   <property name="text">
    <string>Compiler Settings...</string>
   </property>
  </action>
  <action name="actionUnpackage_Document">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unpackage Document...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionMinimize_window">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+F</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Maximize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionMinimize_Screen">
   <property name="text">
    <string>Minimize Screen</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+M</string>
   </property>
  </action>
  <action name="actionLaTeX_Office_on_the_Web">
   <property name="text">
    <string>LaTeX Office on the &amp;Web</string>
   </property>
  </action>
  <action name="actionRun_Slide_Show">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/menu/slideshow.png</normaloff>:/images/menu/slideshow.png</iconset>
   </property>
   <property name="text">
    <string>Run Slide Show...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionRemote_Control_Settings">
   <property name="text">
    <string>Remote Control Settings...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DocumentViewer</class>
   <extends>QWidget</extends>
   <header>documentviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
